web/js/transcribe.js

```diff
@@ -1302,7 +1305,7 @@ function activateUserTools(tools, permissions) {
             '</div>');
         // $('#'+name+"Split").show();
     };
-    if ((tpen.user.isAdmin || permissions.allow_public_modify || permissions.allow_public_modify_line_parsing) && $.inArray("parsing", tools) > -1) {
+    if ((tpen.user.isMember || permissions.allow_public_modify || permissions.allow_public_modify_line_parsing) && $.inArray("parsing", tools) > -1) {
```

Comment:
how did this get so convoluted? It definitely hasn't always been this way, but I also understand what they are trying to do here.